XDS_PROTOCOL.md

```diff
@@ -191,11 +191,15 @@ drop can't be tolerated, traffic drop could have been avoided by providing a
 CDS/EDS update with both __X__ and __Y__, then the RDS update repointing from
 __X__ to __Y__ and then a CDS/EDS update dropping __X__.
 
-In general, to avoid traffic drop:
-* Sequencing should be make before break.
-* LDS and CDS updates should arrive before the respective RDS and EDS updates.
-* CDS/EDS resources corresponding to routes in LDS/RDS should be available at
-  update.
+In general, to avoid traffic drop, sequencing of updates should follow a
+`make before break` model, wherein
+* CDS updates (if any) must always be pushed first.
+* EDS updates (if any) must arrive after CDS updates for the respective clusters.
+* LDS updates must arrive after corresponding CDS/EDS updates.
+* RDS updates related to the newly added listeners must arrive in the end.
+
+xDS updates can be pushed independently if no new clusters/routes/listeners
+are added.
```

Comment:
or if it's acceptable to temporarily drop traffic during updates (this is after all the whole point of the eventual consistency model, some folks don't mind dropping traffic or inconsistent routing across nodes if it makes management simpler).